src/pages/SignUp/PopupSignupSuccess/PopupSignupSuccess.tsx

```diff
@@ -0,0 +1,33 @@
+import InfoImage from '../../../components/InfoImage/InfoImage';
+import styles from './PopupSignupSuccess.module.scss';
+import { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import InfoPopup from '../../../components/Popups/InfoPopup/InfoPopup';
+
+interface PopupSignupSuccessProps {
+    /**
+     * Close window
+     */
+    closeInfoPopup: () => void;
```

Comment:
Здесь и в InfoPopup

```suggestion
    /**
     * Fires when info popup closes
     */
    onCloseInfoPopup: () => void;
```
эвент хэндлеры в пропсах начинаются на on. Как например, onClick, onKeypress. А в доке пишут когда этот хэндлер вызывается.

Reply:
Хорошо. Только можно fires на works?

Reply:
works я бы понял как то, что оно вызывается много раз в течение жизни. Про эвенты, колбэки обычно пишут fires, triggered, is called

Reply:
хорошо, пусть стреляет